predictor/views_utils.py

```diff
@@ -6,3 +6,24 @@ def make_prediction(input_data):
     sh.predict(input_data)
     sh.test()
     return sh.predictions
+
+
+def check_content(necessary_fields, have_fields):
+    missing_context = {}
+
+    for field in necessary_fields:
+        if field not in have_fields:
+            missing_context['no_' + field] = True
+
+    return missing_context
+
+
+# todo change to more valid check
+def validate_email(email):
```

Comment:
**(optionally):** Maybe you should to rename this function to except collision with function from **django.core.validators**?